### DISCUSSION TITLE
QTable - change row style dinamically

### DISCUSSION THREAD

**Discussion by BalazsMakrai** [quasarframework/quasar#14898] [General - Components / Directives / etc] [answered]
Is there any way to set the custom background color for a row in QTable? I want to change the color of the particular row based on props. 
Maybe I should use a `v-slot`, but I can not find the proper one.

**Comment by metalsadman** [comment, marked as answer]
see this example https://quasar.dev/vue-components/table#body-slots

https://codepen.io/metalsadman/pen/MWWZpzG
